draft-ietf-cats-framework.md

```diff
@@ -386,6 +386,30 @@ The "underlay infrastructure" in {{fig-cats-components}} indicates an IP and/or
 
 The following subsections provide an overview of a typical CATS workflow. In order to enable CATS in a given domain, some provisioning is needed; see more details in {{sec-cats-provisioning}}. {{sec-cats-deployment}} describes several deployment options (distributed, centralized, and hybrid model) to accommodate a variety of contexts.
 
+~~~ aasvg
```

Comment:
The figure may be interpreted as if we assume a sequentiality in the various state, which may not be always true. For example service announcement can be updated/modified after metric are distributed.

I think that I prefer to not include this figure.

Reply:
@dzpzp please take a look